arch/arm/dts/imx7ulpea-ucom-kit_v2.dts

```diff
@@ -112,6 +111,12 @@
 			>;
 		};
 
+		pinctrl_system_reset: system-reset {
+			fsl,pins = <
+				IMX7ULP_PAD_PTC10__PTC10	0x20100
```

Comment:
Let's set this to: 0x20000 as the driver won't set mux mode from this value (it's masked).